_docs/dnastack-launch-with.md

```diff
@@ -44,3 +44,5 @@ If not logged into DNAstack, you will be prompted to login. Otherwise or after l
 You will need to pick a version of your workflow to import and a project to import it into.
 Then hit the button to "Import" and continue from within the DNAstack interface to run your workflow. 
 Note that as with the above approach, you will want to double-check that the workflow specifies a runtime environment (docker, cpu, memory, and disks) if you have trouble importing the workflow and that the workflow has not been imported before. 
+
+Additionally, DNAStack does not currently support HTTP or file-based imports.  Importing a workflow with those imports will result in error.
```

Comment:
This doesn't really belong in `## While browsing Dockstore` since it could occur with both scenarios. 
Please elaborate by showing an example of both kind of import. This can be part of a new limitations section.